src/lib/__html.js

```diff
@@ -31,7 +31,7 @@ export default ({
             <script>
             window.gu = {
                 app: {
-                    state: ${JSON.stringify(state)},
+                    app: ${JSON.stringify(data)},
```

Comment:
Shall we call this field `data` for consistency?

Reply:
`app.app`?

Reply:
Yes definitely, that was missed...